src/main/java/econo/buddybridge/post/entity/Post.java

```diff
@@ -0,0 +1,81 @@
+package econo.buddybridge.post.entity;
+
+
+import econo.buddybridge.member.entity.Member;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Table(name = "POST")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@EntityListeners(value = AuditingEntityListener.class)
+public class Post {
+    @Id
+    @Column(name = "post_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+//    @ManyToOne(cascade = CascadeType.REMOVE)
+    @ManyToOne()
+    @JoinColumn(name="author_id")
+    private Member author;
+
+    private String title;
+
+    @Enumerated(EnumType.STRING)
+    private AssistanceType assistanceType;
+
+    @Embedded
+    private DurationPeriod durationPeriod; // startTime & endTime
+
+    @Enumerated(EnumType.STRING)
+    private ScheduleType scheduleType;
+
+    private String scheduleDetail; // 정기 비정기 옆, 상세 정보 입력
```

Comment:
`Schedule` 클래스로 묶는건 어떤가요?

```suggestion
    @Embedded
    private Schedule schedule;
```
```java
@Embeddable
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@EqualsAndHashCode
public class Schedule {

    private LocalDateTime startTime;
    private LocalDateTime endTime;

    @Enumerated(EnumType.STRING)
    private ScheduleType scheduleType;

    private String scheduleDetails;
}
```